spec/rails/doc/smart/expected.yaml

```diff
@@ -83,49 +65,8 @@ paths:
                 properties:
                   message:
                     type: string
-                  no_such_field:
-                    type: string
-                    example: This field does not exist in rspec
               example:
                 message: Unauthorized
-    post:
```

Comment:
Note) `pos` was removed because it matched the pattern `2. paths.*.* (HTTP Methods)`